.github/workflows/native-tests-daily.yml

```diff
@@ -5,9 +5,12 @@ on:
     # daily at 4:00 UTC
     - cron: "0 4 * * *"
   workflow_dispatch:
+  pull_request:
+    types: [ labeled ]
 
 jobs:
-  build:
+  graalvm-native-tests:
+    if: ${{ github.event.label.name == 'native' || github.event.label.name == 'spring boot starter' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'}}
```

Comment:
we have labels like `test openj9` and `test windows`, perhaps `test native` would be better?